src/main/java/com/minecolonies/rankup/modules/core/config/CoreConfig.java

```diff
@@ -14,45 +15,62 @@
 public class CoreConfig extends BaseConfig
 {
     @Setting(value = "welcome-message", comment = "The message to display for a player in chat when they login to the server")
-    public String welcomeMessage = "Welcome back {player}, your current rank is {prefix}!";
+    public String welcomeMessage = "Welcome back " + Constants.PlayerInfo.PLAYER_NAME + ", your current rank is " + Constants.PlayerInfo.PLAYER_PREFIX + "!";
 
     @Setting(value = "first-time-welcome-message", comment = "The message to display for a player in chat when they login to the server for the first time")
-    public String firstWelcomeMessage = "Welcome to our server {player}!";
-
-    @Setting(value = "date-format", comment = "Date format to save data info of players.")
-    public String dateFormat = "dd/MM/yyyy";
+    public String firstWelcomeMessage = "Welcome to our server " + Constants.PlayerInfo.PLAYER_NAME + "!";
 
     @Setting(value = "prefix-fallback", comment = "This is the prefix that will be put in place if a player has no prefix, leave \"\" to have no prefix fall-back")
     public String prefixFallback = "Default";
 
-    @Setting(value = "rankup-command", comment = "The command the plugin will execute via console to add a rank to a player, valid entries: {player} and {group}")
-    public String rankupCommand = "luckperms user {player} parent add {group}";
+    @Setting(value = "rankup-command", comment = "The command the plugin will execute via console to add a rank to a player, valid entries: " + Constants.PlayerInfo.PLAYER_NAME + " and " + Constants.PlayerInfo.PLAYER_GROUP + "")
+    public String rankupCommand = "luckperms user " + Constants.PlayerInfo.PLAYER_NAME + " parent add " + Constants.PlayerInfo.PLAYER_GROUP + "";
 
-    @Setting(value = "rankdown-command", comment = "the command the plugin will execute via console to remove a rank from a player, valid entries: {player} and {group}")
-    public String rankdownCommand = "luckperms user {player} parent remove {group}";
+    @Setting(value = "rankdown-command", comment = "the command the plugin will execute via console to remove a rank from a player, valid entries: " + Constants.PlayerInfo.PLAYER_NAME + " and " + Constants.PlayerInfo.PLAYER_GROUP + "")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 235 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00103)